File: Src/java/engine/src/test/java/org/opencds/cqf/cql/engine/execution/Issue39.java
package org.opencds.cqf.cql.engine.execution;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

import org.junit.jupiter.api.Test;

class Issue39 extends CqlTestBase {

    @Test
    void interval() {
        var results = engine.evaluate(toElmIdentifier("Issue39"));
        Object value = results.forExpression("EquivalentIntervals").value();
        assertThat(value, is(true));
    }
}

File: Src/java/engine/src/test/kotlin/org/opencds/cqf/cql/engine/execution/Issue39.kt
package org.opencds.cqf.cql.engine.execution

import org.hamcrest.MatcherAssert
import org.hamcrest.Matchers
import org.junit.jupiter.api.Test

internal class Issue39 : CqlTestBase() {
    @Test
    fun interval() {
        val results = engine.evaluate(toElmIdentifier("Issue39"))
        val value = results.forExpression("EquivalentIntervals").value()
        MatcherAssert.assertThat(value, Matchers.`is`(true))
    }
}
